cns/restserver/util.go

```diff
@@ -201,7 +206,34 @@ func (service *HTTPRestService) getNetworkContainerResponse(req cns.GetNetworkCo
 		}
 
 		logger.Printf("pod info %+v", podInfo)
-		containerID = service.state.ContainerIDByOrchestratorContext[podInfo.PodName+podInfo.PodNamespace]
+
+		var (
+			context   = podInfo.PodName + podInfo.PodNamespace
+			dncEP     = service.GetOption(acn.OptPrivateEndpoint).(string)
+			infraVnet = service.GetOption(acn.OptInfrastructureNetwork).(string)
+			nodeID    = service.GetOption(acn.OptNodeID).(string)
+			managed   = service.GetOption(acn.OptManaged).(bool)
```

Comment:
can we do this one time..when service starts and save it in memory..instead of doing at multiple places

Reply:
the options are set after HTTPRestService has been init